its/ruling/src/test/resources/guava/squid-S2325.json

```diff
@@ -1,72 +1,20 @@
 {
-'com.google.guava:guava:src/com/google/common/base/Absent.java':[
-101,
```

Comment:
This should be part of another commit... In fact I don't understand how you spotted them.

I know that the `readResolve` method from serialization can not be static, and raising issues on them should be removed, what triggered the fix? Was `INSTANCE` not resolved ?!

Which make me think that we do not have a rule for unknown identifiers in the Semantic Test. We should most probably add one... WDYT?